orientation_readme.md

```diff
@@ -0,0 +1,39 @@
+# Orientation : Welcome to your first day!
+This document was made to help newcomers set up all the software that they will need to actively learn and participate in the JustAskDavidB Coaching Program.
+
+##  Downloading and Setting Up Software
+
+### [Jetbrain](https://account.jetbrains.com/login)
+We use Jetbrain products for web development.  This includes PHPStorm for PHP development, DataGrip for SQL and database interaction, and Pycharm for Python development. Downloading the trials is fine because a JetBrain license will be purchased for you as part of the JustAskDavidB Coaching Program. You will receive an email from JetBrains to complete your registration within the first 15-days. After logging into JetBrains, “Authorization Successful” should appear on screen.
+
+#### 1. [PHPStorm](https://www.jetbrains.com/phpstorm/download/#section=windows)
+Download the program for your OS and run through the installation. After installing Pycharm, open Pycharm and run through the setup wizard. Do not worry there are no wrong answers :)
+
+#### 2. [DataGrip](https://www.jetbrains.com/datagrip/download/#section=windows)
+Download the default “.exe” version. Run the datagrip-2023.2.1.exe file that starts the Installation Wizard. Follow all the steps suggested by the wizard. Pay special attention to the corresponding installation options. 
+
+#### 3. [PyCharm](https://www.jetbrains.com/pycharm/download)
+Download the default “.exe” version. Open the file and set up and install PyCharm. After setting up a trial on Jetbrain switch back to PyCharm and click the blue “Activate” button that has now appeared. Then click “Continue” at the bottom right-hand corner of the screen. PyCharm is now available to be used. 
+
+### [Docker Desktop](https://www.docker.com/)
+Click the blue “Download Docker Desktop” button. Save it and then open “Docker Desktop Installer” where it was saved, then allow the app to make changes to your device. Follow the instructions on the installer to install Docker. 
```

Comment:
Delete "Save it" and combine the first two sentences. 
Delete "where it was saved"
Make "allow the app to make changes to your device." it's own sentence.
Delete "to install Docket"